deployment/roles/profiled-traffic-generator/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+# tasks file for profiled-traffic-generator
+- name: Creates tmp traffic directory
+  tempfile:
+    state: directory
+  register: tempdir_1
+
+- name: Creates traffic directory
+  file:
+    path: "{{ injector_traffic_path }}"
+    state: directory
+
+- name: Run Buddhi to create traffic file
+  docker_container:
+    name: buddhi
+    state: started
+    user: root
+    tty: True
+    detach: no
+    image: quay.io/3scale/perftest-toolkit:master
```

Comment:
Same comment related to version used by users. Do we consider master acceptable for users?

Reply:
same answer :)